src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, FC } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { User } from '../../types';
+
+import {
+  getUserId,
+  isUserError,
+  setUserSelected,
+  getUserFromServer,
+  getUserInfo,
+  setUser
+} from '../../store';
+import './CurrentUser.scss';
+
+export const CurrentUser: FC = () => {
+
+  const dispatch = useDispatch();
+
+  const user: User = useSelector(getUserInfo);
+  const userId: number = useSelector(getUserId);
+  const isUserLoadError: boolean = useSelector(isUserError);
+
+  const fetchUserData = () => {
+    return dispatch(getUserFromServer(userId));
+  };
+
+  useEffect(() => {
+    if (userId && userId > 0) {
+      fetchUserData();
+    }
+  }, [userId]);
+
+  const clearUser = () => {
+    dispatch(setUser({}));
```

Comment:
```suggestion
    dispatch(setUser(null));
```
Если нужно убрать значение - устанавливай null